### PR TITLE
add dependency and remove wrong unittest

### PR DESCRIPTION
**Pull request by ydailinkedin**

### CODE REVIEW

File: camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/CamusJobTestWithMock.java [hunk 1]
```diff
@@ -274,7 +274,6 @@ public void testJobFailDueToOffsetRangeCallError() throws Exception {
     setupJobFailDueToOffsetRangeCallError();
     job = new CamusJob(props);
     job.run(EtlInputFormatForUnitTest.class, EtlMultiOutputFormat.class);
-    verifyJobSucceed();
```
Comment (HenryCaiHaiying) [inline camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/CamusJobTestWithMock.java]: Why is this removed?

Comment (zliu41) [inline camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/CamusJobTestWithMock.java]: This job is supposed to fail, so should not verify job succeed. 
This somehow passes on maven, but fails on ligradle.

Comment (HenryCaiHaiying) [inline camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/CamusJobTestWithMock.java]: OK, I see.
